app/templates/gentelella/users/events/_table.html

```diff
@@ -53,15 +53,15 @@
 
                     </td>
                     <td>
-                        <ol style="padding-left:0;">
+                        <ol style="padding-left:10px;">
                             <li>{{ _("Drafts") }}: {{ drafts.count }}</li>
                             <li>{{ _("Submitted") }}: {{ event.session | length }}</li>
                             <li>{{ _("Accepted") }}: {{ accepted.count }}</li>
                             <li>{{ _("Pending") }}: {{ pending.count }}</li>
                             <li>{{ _("Rejected") }}: {{ rejected.count }}</li>
                         </ol>
                     </td>
-                    <td>{{ event.speaker | length }}</td>
+                    <td style="padding-left:30px;">{{ event.speaker | length }}</td>
```

Comment:
@Himanshi-Khandelwal then check what is causing the mis-alignment. The issue could be something that is common to all tables. And if you're able to figure it out and fix it for all tables in one go it'll be great